odoo/addons/base/data/res.country.state.csv

```diff
@@ -1382,3 +1382,27 @@ state_no_46,base.no,"Vestland","NO-46"
 state_no_30,base.no,"Viken","NO-30"
 state_no_22,base.no,"Jan Mayen","NO-22"
 state_no_21,base.no,"Svalbard","NO-21"
+state_ec_1,ec,"Azuay","AZUAY"
```

Comment:
the third column is the code which is usually a 2 letters abreviation, don't you have that? Also could you add that directly in `l10n_ec` instead?